### ISSUE TITLE
Mismatching Logic in Train and Evaluation

### ISSUE THREAD

**Issue by edenpelleg** [itaigat/pytorch-template#2]
Hey Itai,

We noticed that in your training you multiply the loss by the size of the batch, whereas in the evaluation you do not.
Is it missing or are we missing something?

Thanks!

Train:
https://github.com/itaigat/pytorch-template/blob/1bed9f0e90a03295826c86ce2ea2826dc7a53de2/train.py#L78

Evaluation:
https://github.com/itaigat/pytorch-template/blob/1bed9f0e90a03295826c86ce2ea2826dc7a53de2/train.py#L136

**Comment by itaigat** [comment]
You are correct, fixed it. Thanks!

* note that it's only for logging and does not related to the training procedure
